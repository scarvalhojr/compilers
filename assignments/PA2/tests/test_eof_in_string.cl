CLASS Foo {
    bar : String <- "EOF in string constant