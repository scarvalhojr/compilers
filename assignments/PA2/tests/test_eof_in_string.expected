#name "tests/test_eof_in_string.cl"
#1 CLASS
#1 TYPEID Foo
#1 '{'
#2 OBJECTID bar
#2 ':'
#2 TYPEID String
#2 ASSIGN
#2 ERROR "EOF in string constant"
